core/src/test/java/org/apache/iceberg/hadoop/TestHadoopCatalog.java

```diff
@@ -284,18 +285,27 @@ public void testDropNamespace() throws IOException {
     Configuration conf = new Configuration();
     String warehousePath = temp.newFolder().getAbsolutePath();
     HadoopCatalog catalog = new HadoopCatalog(conf, warehousePath);
+    Namespace namespace1 = Namespace.of("db");
+    Namespace namespace2 = Namespace.of("db", "ns1");
 
-    TableIdentifier tbl1 = TableIdentifier.of("db", "tbl1");
-    TableIdentifier tbl2 = TableIdentifier.of("db1", "ns1", "tbl1");
+    TableIdentifier tbl1 = TableIdentifier.of(namespace1, "tbl1");
+    TableIdentifier tbl2 = TableIdentifier.of(namespace2, "tbl1");
 
     Lists.newArrayList(tbl1, tbl2).forEach(t ->
         catalog.createTable(t, SCHEMA, PartitionSpec.unpartitioned())
     );
 
+    AssertHelpers.assertThrows("Should fail to drop namespace is not empty" + namespace1,
```

Comment:
This needs a space after "empty" or else the namespace will not be separated.

Reply:
OK thanks